### PR TITLE
WSTEAMA 487 Launch Route Burmese to Homepage on Live Environment

### PR DESCRIPTION
**Pull request by holchris**
Resolves JIRA [478](https://jira.dev.bbc.co.uk/browse/WSTEAMA-487)

Overall changes
======
Updates the logic to ensure that on all environments - local, test, live that the homepage is rendered when service=burmese

Code changes
======
- Updated test so /burmese renders a homepage, not a frontpage
- Updated logic so that /burmese renders a homepage not a frontpage 
- Removes /burmese from cypress test for frontpage on live environment by removing configuration

Testing
======

Helpful Links
======

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/components/MostRead/index.stories.tsx [hunk 1]
```diff
@@ -2,31 +2,31 @@ import React from 'react';
 import { withKnobs } from '@storybook/addon-knobs';
 import metadata from './metadata.json';
 import md from './README.md';
+import MostRead from '.';
```
Comment (karinathomasbbc) [inline src/app/components/MostRead/index.stories.tsx]: Oops I think this commit is on the wrong branch...

**Review by karinathomasbbc — DISMISSED** [review]
We also need to make the routing changes in the regex files to ensure that burmese is routed to the new homepahe

**Review by LilyL0u — APPROVED** [review]
it good

**Review by Isabella-Mitchell — APPROVED** [review]
Cool, I compared it to the Kyrgyz front page and the methodolgy looks the same.

**Pull request merged**